app/build.gradle

```diff
@@ -30,8 +30,10 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
-    implementation 'androidx.appcompat:appcompat:1.0.2'
-    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.0-beta8'
+    implementation 'com.google.android.material:material:1.1.0'
+    implementation files('libs/commons-lang3-3.10.jar')
```

Comment:
Don't include the file. 
https://mvnrepository.com/artifact/org.apache.commons/commons-lang3/3.10